### PR TITLE
Change name normalize() to min_max_scaling

### PR DESCRIPTION
**Pull request by jenniew**
Change normalize() to min_max_scaling(). 

To support inference, the API may change to scaler = tbl.gen_min_max_scaler(cols), tbl.min_max_scaling(cols, scaler)
Or tbl, min_max_array = tbl.min_max_scaling(cols)?

### CODE REVIEW

**Comment by jason-dai** [comment]
`min_max_scale`

**Comment by hkvision** [comment]
Change the output to be: mapping = {c: (self[c].min(), self[c].max()) for c in columns} to align with the dummy pipeline

**Comment by jenniew** [comment]
Jenkins: http://10.239.176.111:18888/job/ZOO-PR-Validation/6064/

File: pyzoo/zoo/friesian/feature/table.py [hunk 1]
```diff
@@ -844,7 +844,7 @@ def cross_columns(self, crossed_columns, bucket_sizes):
         df = cross_columns(self.df, crossed_columns, bucket_sizes)
         return FeatureTable(df)
 
-    def normalize(self, columns):
+    def min_max_scale(self, columns):
         """
         Normalize numeric columns
```
Comment (hkvision) [inline pyzoo/zoo/friesian/feature/table.py:849]: change this doc. At least describe how to normalize

Comment (jenniew) [reply pyzoo/zoo/friesian/feature/table.py]: updated

**Pull request merged**